propelauth_py/api/token_verification_metadata.py

```diff
@@ -35,5 +38,5 @@ def _fetch_token_verification_metadata(
     json_response = response.json()
     return TokenVerificationMetadata(
         verifier_key=json_response.get("verifier_key_pem"),
-        issuer=auth_url,
+        issuer="https://" + auth_hostname,
```

Comment:
need to make sure the expected issuer is still the auth_url so the jwt validation succeeds